src/app/legacy/containers/PodcastExternalLinks/index.jsx

```diff
@@ -122,44 +143,42 @@ const PodcastExternalLinks = ({ brandTitle, links }) => {
       </ThirdPartyLinksTitle>
       {hasMultipleLinks ? (
         <StyledList role="list">
-          {links.map(({ linkText, linkUrl }) => (
+          {links.map(({ linkText, linkUrl, linkType }) => (
             <StyledListItem dir={dir} key={`${linkText}`}>
-              <Link
-                // line 126 and id={`externalLinkId-${linkText}`} in line 133 are a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
-                aria-labelledby={`externalLinkId-${linkText}`}
-                href={linkUrl}
-                service={service}
-                script={script}
-                dir={dir}
-                onClick={clickTrackerRef}
+              <PodcastExternalLink
+                linkText={linkText}
+                linkUrl={linkUrl}
+                linkType={linkType}
+                brandTitle={brandTitle}
+                aria={{ 'aria-labelledby': `externalLinkId-${linkText}` }}
```

Comment:
May want to keep this comment, as it's still a temp fix
```
// line 126 and id={`externalLinkId-${linkText}`} in line 133 are a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
```

Reply:
comment re-added in this PR: https://github.com/bbc/simorgh/pull/10544